backend/benefit/applications/management/commands/send_ahjo_requests.py

```diff
@@ -248,6 +248,14 @@ def _handle_successful_request(
     ) -> None:
         if request_type == AhjoRequestType.GET_DECISION_DETAILS:
             response_handler = AhjoDecisionDetailsResponseHandler()
+
+            if not response_content:
+                self._handle_failed_request(
+                    counter, application, request_type, "No details found in response."
+                )
```

Comment:
Can leave this out IMO, or replace it with something else. The request itself doesn't fail, it's just that the response is empty, i.e. the decision details weren't found.